templates/base.html

```diff
@@ -29,12 +29,6 @@
               <div class="hxRow">
                 <nav class="hxBreadcrumb">
                   <a style="cursor:default;">{{page.category}}</a>
```

Comment:
(note) should make sure that the `cursor: default;` override is ported into HelixUI, so that you don't have to override it here

Reply:
👍